libraries/AC_Avoidance/AC_Avoid.h

```diff
@@ -41,17 +41,17 @@ class AC_Avoid {
     // return true if any avoidance feature is enabled
     bool enabled() const { return _enabled != AC_AVOID_DISABLED; }
 
-    /*
-     * Adjusts the desired velocity so that the vehicle can stop
-     * before the fence/object.
-     * Note: Vector3f version is for convenience and only adjusts x and y axis
-     */
-    void adjust_velocity(float kP, float accel_cmss, Vector2f &desired_vel_cms, float dt) {
+    // Adjusts the desired velocity so that the vehicle can stop
```

Comment:
let's add some comments here to clarify that kP and accel_cmss are for the horizontal axis.  "kP" by the way is the gain controlling the position error to desired speed.